tests/components/hdmi_cec/__init__.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for the HDMI-CEC component."""
+from unittest.mock import AsyncMock, Mock
+
+from homeassistant.components.hdmi_cec import KeyPressCommand, KeyReleaseCommand
+
+
+class MockHDMIDevice:
+    """Mock of a HDMIDevice."""
+
+    def __init__(self, *, logical_address, **values):
+        """Mock of a HDMIDevice."""
+        super().__setattr__(
```

Comment:
Yeah, this is a bit weird, I agree. 

This is because of the [`__setattr__` overwrite in line 30](https://github.com/home-assistant/core/pull/75094/files/42cd37372ceda9feffd1ed64fa5b923360a0fa98#diff-f30a2ae42f38450385c5471b7ed02b615dc3a354a953863b2415a79f6cc9e54aR30-R35). which just puts the values in the `_values` dict. This make the testing easy & makes it so that we don't have to think about which attributes the class needs to have and set. But I think it does make the code less readable, I'll clean it up :)